From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MiniDigger | Martin <admin@benndorf.dev>
Date: Sun, 31 Mar 2024 17:01:32 +0200
Subject: [PATCH] Base


diff --git a/net/minecraft/CrashReport.java b/net/minecraft/CrashReport.java
index 7efc2d1a901ea5708045fc9fddca09caf467762f..659b933e4c077cb6b36c23e9844e314ecb2f1110 100644
--- a/net/minecraft/CrashReport.java
+++ b/net/minecraft/CrashReport.java
@@ -29,7 +29,7 @@ public class CrashReport {
     private Path saveFile;
     private boolean trackingStackTrace = true;
     public StackTraceElement[] uncategorizedStackTrace = new StackTraceElement[0];
-    public SystemReport systemReport = new SystemReport();
+    private SystemReport systemReport = new SystemReport(); // woooo at base test???? who knew
 
     public CrashReport(String title, Throwable exception) {
         this.title = title;
