From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MiniDigger | Martin <admin@benndorf.dev>
Date: Sun, 31 Mar 2024 17:01:32 +0200
Subject: [PATCH] Base


diff --git a/net/minecraft/CrashReport.java b/net/minecraft/CrashReport.java
index 01c9784075b31876a0a91e81df0ae50a42d246e3..25837ee13ac83e64c3bea23f5abf4e6dd1e39829 100644
--- a/net/minecraft/CrashReport.java
+++ b/net/minecraft/CrashReport.java
@@ -29,7 +29,7 @@ public class CrashReport {
     private @Nullable Path saveFile;
     private boolean trackingStackTrace = true;
     public StackTraceElement[] uncategorizedStackTrace = new StackTraceElement[0];
-    public SystemReport systemReport = new SystemReport();
+    private SystemReport systemReport = new SystemReport(); // woooo at base test???? who knew
 
     public CrashReport(final String title, final Throwable t) {
         this.title = title;
