From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MiniDigger | Martin <admin@benndorf.dev>
Date: Sun, 20 Oct 2024 09:24:36 +0200
Subject: [PATCH] Feature


diff --git a/Test.java b/Test.java
index b5106a3b36987dfdd5141ae1cbd229e6db020cd9..412773ad543cc9a5116044dc138cc08300b2dd1b 100644
--- a/Test.java
+++ b/Test.java
@@ -13,4 +13,8 @@ public class Test {
     public final String getTest2() {
         return this.test + "2";
     }
+
+    public String feature() {
+        return "amazing feature";
+    }
 }
