From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MiniDigger | Martin <admin@benndorf.dev>
Date: Sun, 20 Oct 2024 09:24:36 +0200
Subject: [PATCH] Feature


diff --git a/Test.java b/Test.java
index ec4473011075a4028645a320ed777c8adce0f77d..55fb66e3462ad6acfb7054f416f07d142672b92c 100644
--- a/Test.java
+++ b/Test.java
@@ -13,4 +13,8 @@ public class Test {
     public String getTest2() {
         return this.test + "2"; // Woo
     }
+
+    public String feature() {
+        return "amazing feature";
+    }
 }
